polaris-react/src/components/Tooltip/Tooltip.scss

```diff
@@ -0,0 +1,3 @@
+.TooltipContainer {
```

Comment:
I think we'll want a comment here if we go this route [cause lint ](https://github.com/Shopify/polaris/actions/runs/3906349172/jobs/6674426119)will be 😠 hehe: 

<img width="897" alt="Screenshot 2023-01-12 at 5 18 36 PM" src="https://user-images.githubusercontent.com/43223543/212214958-1291cfa0-d14d-4531-a7af-1df375033538.png">